### DISCUSSION TITLE
Account Activation

### DISCUSSION THREAD

**Discussion by quinderbi** [codalab/codabench#1406] [Q&A]
I am writing to report an issue I encountered while attempting to reactivate my account. I have faced significant difficulties, and I hope you can assist me in resolving this matter promptly.

Initially, I tried to log into my account but failed due to an activation problem. Subsequently, I attempted to register again using the same email and username, only to receive a message stating that the email and username were already registered in the system.

I then tried to reactivate my account by following the activation link sent via email. Unfortunately, upon clicking the link, I discovered that it had already expired.
<img width="1440" alt="Screenshot 2024-04-12 at 08 07 37" src="https://github.com/codalab/codabench/assets/101580458/4098b6db-0bfa-4cc9-bc5a-5c1715d8a81f">
<img width="1440" alt="Screenshot 2024-04-12 at 08 08 28" src="https://github.com/codalab/codabench/assets/101580458/5c883fd1-b1be-465d-880d-f9ca029bf610">
<img width="1440" alt="Screenshot 2024-04-12 at 08 08 41" src="https://github.com/codalab/codabench/assets/101580458/9b178f78-1186-445b-aace-645d292dc723">

**Comment by Didayolo** [comment]
I manually activated your account. Please confirm if it worked.

This problem is related to this issue:
- #1018
